frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -384,6 +422,34 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
         sessionPlayerSnapshotData: [
             null as SessionPlayerSnapshotData | null,
             {
+                pollRecordingSnapshots: async (_, breakpoint: BreakPointFunction) => {
+                    const params = toParams({
+                        source: 'realtime',
+                        version: '2',
+                    })
+
+                    await breakpoint(1)
```

Comment:
```suggestion
```

Reply:
https://keajs.org/docs/core/listeners/#breakpoints

this is a debounce... tbh there should probably be `breakpoint()`  after the API call to account for slow network responses too

Reply:
lol, There are only two hard things in Computer Science: cache invalidation and naming things.
https://martinfowler.com/bliki/TwoHardThings.html
I thought it was a "PLEASE STOP DEBUGGER HERE RIGHT NOW"